debian/packaging-debian.md

```diff
@@ -0,0 +1,37 @@
+# Packaging for Debian
+
+Packaging sources for Debian are contained in the
+[`debian` branch](https://github.com/awslabs/amazon-ecr-credential-helper/tree/amazonlinux).
+
+This branch is updated every time a release is prepared for Debian.
+
+To prepare a release, you can use the following workflow:
+
+* Merge the release tag into this branch
+* Delete any files that have been added to the
```

Comment:
can we programmatically check that this is empty?

Reply:
A good idea, but not going to do it as part of this PR.